CWE/decisions/italy.txt

```diff
@@ -0,0 +1,95 @@
+political_decisions = {
+
+	ina_casa = {
+		picture = "piano_fanfani"
+		potential = {
+		    year = 1949
```

Comment:
Please follow the coding convention of:

TAG
Year
Government 
Vassal yes or no
War 
Ruling party
Money
Provinces owned
Flags